templates/dashboard/base.html

```diff
@@ -165,7 +165,7 @@
             {% if user.is_superuser %}
```

Comment:
Here we should have check for specific permissions (just like in previous sections) and not check for superuser flag.

Reply:
Ok, changed.